packages/ibm-products/src/components/Tearsheet/TearsheetShell.stories.jsx

```diff
@@ -140,3 +142,72 @@ export const NoAttributesSet = Template.bind({});
 NoAttributesSet.args = {
   size: 'wide',
 };
+
+function FloatingMenu({ open, className, styleTransform }) {
+  return (
+    open &&
+    createPortal(
+      <div
+        className={className}
+        style={{
+          position: 'absolute',
+          top: '25%',
+          left: '50%',
+          transform: styleTransform,
+          border: 'solid 1px',
+          zIndex: '9999',
+          backgroundColor: 'white',
+        }}
+      >
+        <TextInput
+          type="text"
+          style={{ margin: '1rem' }}
+          labelText={`This is a floating DOM element with classname "${className}" added outside of the tearsheet DOM element`}
+          placeholder="Focus and enter something"
+        ></TextInput>
+      </div>,
+      document.body
+    )
+  );
+}
+
+export const FocusOnFloatingMenu = ({ open: _open, ...args }, context) => {
```

Comment:
Here is the recording showcasing the new property in action.

https://github.com/carbon-design-system/ibm-products/assets/38996291/6730b015-88fb-4f72-becb-12ad06391708

Reply:
It works fine in story `docs` as well